src/core/zowe/core_for_zowe_sdk/sdk_api.py

```diff
@@ -11,51 +11,49 @@
 """
 
 import urllib
-from .logger import Log
 
 from . import session_constants
+from .logger import Log
 from .request_handler import RequestHandler
 from .session import ISession, Session
-from .logger import Log
 
 
 class SdkApi:
     """
     Abstract class used to represent the base SDK API.
     """
 
-    def __init__(self, profile, default_url, logger_name = __name__):
-        self.profile = profile
+    def __init__(self, profile, default_url, logger_name=__name__):
         session = Session(profile)
         self.session: ISession = session.load()
 
         self.logger = Log.registerLogger(logger_name)
 
-        self.default_service_url = default_url
-        self.default_headers = {
+        self._default_service_url = default_url
+        self._default_headers = {
             "Content-Type": "application/json",
             "X-CSRF-ZOSMF-HEADER": "",
         }
 
-        self.request_endpoint = session.host_url + self.default_service_url
+        self._request_endpoint = session.host_url + self._default_service_url
 
-        self.request_arguments = {
-            "url": self.request_endpoint,
-            "headers": self.default_headers,
+        self._request_arguments = {
+            "url": self._request_endpoint,
+            "headers": self._default_headers,
```

Comment:
I noticed that a couple of these variables are prefixed with a single underscore as opposed to double-underscore. I don't have a preference for one style over the other, but am curious to know if we want to make the style consistent for all private variables?

Reply:
The variables with single underscore indicate that we should avoid using them outside, but since some other files use them anyhow, I put them in a less strict private mode so that those files can refer to them. On the other hand, the double underscore private variables are not used outside of the original file, hence I can put them as strict private.